### ISSUE TITLE
Version Drift

### ISSUE THREAD

**Issue by Eonasdan** [leeoniya/dropcss#60]
Hi there, great tool. Thanks for your hard work on this. I use this in my static site generator.

I was doing some updates and noticed that the npm package is 2 years old. Is there much version drift between the published package and github?

Many thanks.

**Comment by leeoniya** [comment]
no, there have been a couple bug fixes, but the main breaking change is https://github.com/leeoniya/dropcss/issues/46

you can see here havent been many commits since the published version: https://github.com/leeoniya/dropcss/commits/master

i'll likely publish a new release in the next few weeks.
